docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -650,11 +650,28 @@ To work around this problem, you can reset the {ecloud} agent policy with an API
 curl -u elastic:<password> --request POST \
   --url <kibana_url>/internal/fleet/reset_preconfigured_agent_policies/policy-elastic-agent-on-cloud \
   --header 'Content-Type: application/json' \
-  --header 'kbn-xsrf: xyz' \
+  --header 'kbn-xsrf: xyz'
 ----
 
 [discrete]
 [[pgp-key-download-fail]]
 == Air-gapped {agent} upgrade can fail due to an inaccessible PGP key
 
 In versions 8.9 and above, an {agent} upgrade may fail when the upgrader can't access a PGP key required to verify the binary signature. For details and a workaround, refer to the link:https://www.elastic.co/guide/en/fleet/8.9/release-notes-8.9.0.html#known-issue-3375[PGP key download fails in an air-gapped environment] known issue in the version 8.9.0 Release Notes or to the link:https://github.com/elastic/elastic-agent/blob/main/docs/pgp-workaround.md[workaround documentation] in the elastic-agent GitHub repository.
+
+[discrete]
+[[fleet-server-integration-removed]]
+== {agents} unable to connect after removing {fleet-server} integration
+
+When you use {fleet}-managed {agent}, at least one {agent} is running the link:https://docs.elastic.co/integrations/fleet_server[{fleet-server} integration]. In case the policy containing this integration is accidentally removed from {agent], all other agents will not be able to be managed. However, the {agents} will continue to send data to their configured output.
+
+One way to fix this issue is to deploy a new {agent} with the {fleet-server} integration on the same host and port, with TLS certificates that are accepted by the existing {agents}.
+
+To resolve this problem:
+
+. Follow the {agent} <<elastic-agent-installation-steps,installation steps>> as though you are installing a new {agent}. This will give you the CLI arguments that you need.
+
+. Next, you can either:
+
+.. Run the `./elastic-agent enroll` command on the {agent} that previously had the {fleet-server} integration. This effectively returns the agent to use the former {fleet-server} policy.
+.. Run `./elastic-agent install` to reinstall the {agent} from scratch using the former {fleet-server} policy.
```

Comment:
I believe we can clarify more on what needs to be done here and add an example.

Here are some points I believe we can improve:
1. I believe we can make more clear the assumption the Elastic-Agent that was running the Fleet-Server integration is still installed and health, but running another policy
2. To recover that Elastic-Agent, we need to follow the "add new agent" steps in Kibana until we see the CLI command
3. At this point we need to decide what to do:
    - If the "old" Fleet-Server Elastic-Agent is still running and healthy, we need to re-enrol it.
    - If the "old" Fleet-Server Elastic-Agent is not installed any more, we need to install it again.
4. Given Kibana gave us the following command:
    ```
    sudo ./elastic-agent install --url=https://fleet-server:8220 --enrollment-token=bXktc3VwZXItc2VjcmV0LWVucm9sbWVudC10b2tlbg==
    ```
    - If we're re-enrolling the Elastic-Agent:
        - Go to the folder the Elastic-Agent is running (e.g: `/opt/Elastic/Agent/` on Linux) and run (a root):
        ```
        ./elastic-agent enroll --url=https://fleet-server:8220 --enrollment-token=bXktc3VwZXItc2VjcmV0LWVucm9sbWVudC10b2tlbg==
        ```
    - If we're installing a new Elastic-Agent, we just need to follow the steps in Kibana, no changes needed)
        - Download and extract the tar.gz package
        - Run the install command from the extracted folder (follow the steps in Kibana):
        ```
        sudo ./elastic-agent install --url=https://fleet-server:8220 --enrollment-token=bXktc3VwZXItc2VjcmV0LWVucm9sbWVudC10b2tlbg==
        ```